main.py

```diff
@@ -0,0 +1,9 @@
+from uvicorn import run
+from fastapi import FastAPI
+from src.routes.endpoints import router
+
+app=FastAPI()
+
+if __name__ == "__main__":
+    app.include_router(router)
+    run(app, host="0.0.0.0", port=8000)
```

Comment:
these should be env variables, please use pydantic settings as follows:
- under configurations/config.py use pydantic settings to define the host and port of your service there!